dc-cudami-server/dc-cudami-server-backend-jdbi/src/main/java/de/digitalcollections/cudami/server/backend/impl/jdbi/identifiable/IdentifiableRepositoryImpl.java

```diff
@@ -279,7 +274,7 @@ protected Integer selectNextSortIndexForParentChildren(
     return sortIndexDb;
   }
 
-  protected int getIndex(LinkedHashSet<? extends Identifiable> list, Identifiable identifiable) {
+  protected int getIndex(List<? extends Identifiable> list, Identifiable identifiable) {
     int pos = -1;
     for (Iterator iterator = list.iterator(); iterator.hasNext(); ) {
```

Comment:
Könnte man `for (Identifieable value : list)` verwenden.

Reply:
yep